src/api-calls/api-calls.js

```diff
@@ -29,7 +29,7 @@ const getPrescription = (endpoint) => {
 }
 
 const postPrescription = (body) => {
-    return fetch("https://tasterx.herokuapp.com/prescriptions/", {
+    return fetch("https://tasterx.herokuapp.com/presciptions/", {
```

Comment:
```suggestion
    return fetch("https://tasterx.herokuapp.com/prescriptions/", {
```
Looks like this got accidentally changed?